.github/workflows/drivers.yml

```diff
@@ -831,6 +831,7 @@ jobs:
       DRIVERS: redshift
       MB_REDSHIFT_TEST_USER: metabase_ci
       MB_REDSHIFT_TEST_DB: testdb
+      MB_REDSHIFT_TEST_DB_ROUTING: dev
```

Comment:
This would read more clearly to me if it was `MB_REDSHIFT_TEST_ROUTING_DB` but could just be me.